tests/fixtures/test_rst_008.html

```diff
@@ -1,6 +1,6 @@
 <p>Here is some Python code for a <tt>Dog</tt>:</p>
 <pre><span class="k">class</span> <span class="nc">Dog</span><span class="p">(</span><span class="n">Animal</span><span class="p">):</span>
-    <span class="k">def</span> <span class="nf">__init__</span><span class="p">(</span><span class="bp">self</span><span class="p">,</span> <span class="n">name</span><span class="p">):</span>
+    <span class="k">def</span> <span class="fm">__init__</span><span class="p">(</span><span class="bp">self</span><span class="p">,</span> <span class="n">name</span><span class="p">):</span>
```

Comment:
This is due to https://bitbucket.org/birkenfeld/pygments-main/src/05818a4ef9891d9ac22c851f7b3ea4b4fce460ab/CHANGES?at=default&fileviewer=file-view-default#CHANGES-41:42